src/pages/FrontendMentor/KYStatsCard/styles/KYStatsCard.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+
+export const MainPage = styled.div`
+  position: fixed;
+  top: 48px;
+  height: 100%;
+  width: 100%;
+  background-color: hsl(233, 47%, 7%);
+`
+
+export const InformationCard = styled.div`
+  position: fixed;
+  width: 80%;
+  height: 44%;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  background-color: hsl(244, 38%, 16%);
+  border-radius: 10px;
+  display: flex;
+  flex-direction: row-reverse;
+  @media (max-width: 375px) {
+    flex-direction: column;
+    height: 80%;
+  }
+`
+
+export const InformationContainer = styled.div`
+  width: 50%;
+  padding: 50px;
+  @media (max-width: 375px) {
+    width: 100%;
+    text-align: center;
+    padding: 20px;
+  }
+`
+export const ImageContainer = styled.div`
+  width: 50%;
+  background-color: hsl(277, 64%, 61%);
+  @media (max-width: 375px) {
+    width: 100%;
+  }
```

Comment:
Add 10px to border-bottom-right-radius and border-top-right-radius to make the image side border rounded

Reply:
resolved in commit f82d812